README.md

```diff
@@ -17,6 +17,7 @@ $ cp .env.example .env
 #### Building
 
 ```bash
+$ sh run-local.sh
```

Comment:
This isn't needed, `make run` already does it.

Reply:
ok thankyou